src/Microsoft.DotNet.XHarness.Tests.Runners/xUnit/XUnitFilter.cs

```diff
@@ -86,6 +86,23 @@ public static XUnitFilter CreateTraitFilter(string traitName, string traitValue,
             };
         }
 
+        public static XUnitFilter CreateAttributeFilter(string attributeName, bool exclude)
+        {
+            if (string.IsNullOrEmpty((attributeName)))
+            {
+                throw new ArgumentException("must not be null or empty", nameof(attributeName));
```

Comment:
Seems like the perfect place for ArgumentNullException

Reply:
I followed the pattern of the others, I do agree, I'll do a PR setting all of the to do NRE or argument exception accordingly. Added issue: https://github.com/dotnet/xharness/issues/131